mlflow/models/evaluation/default_evaluator.py

```diff
@@ -1818,6 +1824,23 @@ def _log_eval_table(self):
             uri=mlflow.get_artifact_uri(artifact_file_name)
         )
 
+    def _log_genai_custom_metrics(self, genai_custom_metrics):
+        if len(genai_custom_metrics) == 0:
+            return
+
+        data = {"name": [], "version": [], "metric_args": []}
+        for genai_metric_args in genai_custom_metrics:
+            data["name"].append(genai_metric_args["name"])
+            # Custom metrics created from make_genai_metric_from_prompt don't have version
+            data["version"].append(genai_metric_args.get("version", None))
```

Comment:
Something to test - if we store a group of metrics that are all defined via `make_genai_metric_from_prompt` that have `version` of `None`, when this is serialized via `log_table()` (which uses Pandas to write), what is the inferred type of that column and what does deserialization of that data look like? Is it `NaN` ? `<NA>` ?

Reply:
Can we add this exact test case?

Reply:
good catch. Let me add this exact test case

Reply:
I'll pass in empty string "" as placeholder to avoid the none/nan/na behaviour